src/app/cube/home/learning-object-info/learning-outcomes/learning-outcomes.component.scss

```
@@ -1,15 +1,78 @@
-.content{
+@import 'vars.scss';
+
+.content {
     margin-top: 50px;
     margin-right: 50px;
+    padding-left: 0px;
     display: flex;
     flex-direction: column;
-    .taxonomy{
-      margin-bottom: 400px;
+    
+    .blooms-cards {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        gap: 25px;
+        color: $homepage-light-blue;
+        padding-bottom: 50px 0 300px 0;
+
+        .blooms-card {
+            padding: 20px 40px;
```

I would suggest making this a chunkier card to match all the other cards on the page. So I would remove the padding, min/max width, and defined height and replace with: 
```suggestion
           display: flex;
            justify-content: center;
            align-items: center;
            height: 190px;
            width: 265px;
```